eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +51,53 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
-              }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
+    <div key={id + "-" + timeString} className={classes.wrapper}>
```

Comment:
Key is unnecessary here. The key is used by react-fabric (a reconciliation algorithm) to key react nodes in the component tree. It is used to:
- key nodes in the list (remember, you key the nodes, not the elements, i.e. you would do this:
 
```jsx
{map(() => 
  <AttendanceCard key="some_key">
    <Child />
  </AttendanceCard>
}
``` 
 not
```jsx
{map(() => 
  <AttendanceCard>
    <Child key="some_key" />
  </AttendanceCard>
}
``` 
as the `<div>` node you're keying is a child of `<AttendanceCard>` node when you put it all together.
- the other usage (since it is primarily used by react-fiber to recognize what needs to be rerendered), would be to intentionally force rerender of component by changing the key (which is really advanced usage and is rarely used)